.github/ISSUE_TEMPLATE/tracking-issue.md

```diff
@@ -7,13 +7,35 @@ labels: management/tracking, status/proposed
 
 ## Description
 
-<!-- Short description of the proposed feature -->
+Short description of the proposed feature.
 
-## Working Backwards
+## Roles
 
-<!-- Present this feature/change to users as if it was already been implemented -->
+| Role                | User
+|---------------------|------------------------------
+| Proposed by         | @alias
+| Author              | @alias
+| API Bar Raiser      | @alias
+| Stakeholders        | @alias, @alias, @alias
 
-## Roles
+> See [RFC Process](https://github.com/aws/aws-cdk-rfcs#rfc-process) for details
+
+## Workflow
+
+- [x] Tracking issue created (label: `status/proposed`)
+- [ ] API Bar Raiser assigned (ping us at [#aws-cdk-rfcs](https://cdk-dev.slack.com/archives/C025ZFGMUCD) if needed)
+- [ ] Kick off meeting
+- [ ] RFC pull request created (label: `status/review`)
+- [ ] Community reach out (via Slack and/or Twitter)
+- [ ] API Bar Raiser signed-off
```

Comment:
>> How long should these periods be? Worth adding here as a reminder.
>> 
>> One of the items we discussed was making the RFC process quick. I would personally prefer for simple ones to not exceed 5 days or a week.
>
> Not sure how to phrase this here. Maybe we can add some info about periods in the README. Thoughts?

Could be a good start yeah. Each state in the README which needs active waiting (like 'final comments period', 'waiting for feedback', etc.) can have a clear subtext or highlight of on what's the required max waiting period. Some (most?) rfcs may not receive any community input.

Then link each section in the tracking issue to the anchor link in the README?

Reply:
Final comments period already indicates a `~week`. I've added a small note about feedback. I think that's good enough for now.